src/cpu/aarch64/acl_indirect_gemm_convolution.cpp

```diff
@@ -21,8 +21,10 @@ namespace impl {
 namespace cpu {
 namespace aarch64 {
 
-status_t acl_indirect_gemm_convolution_fwd_t::execute_forward(
-        const exec_ctx_t &ctx) const {
+template <data_type_t src_type, data_type_t wei_type, data_type_t dst_type,
```

Comment:
Hi @lilh9598, thanks for your contribution!

I know this is an existing pattern for `acl_gemm_convolution_fwd_t`, but we don't need to template this class, we can handle the different types at runtime like in ACL binary and eltwise. Removing the templating should reduce build time and build size.

The same applies to winograd and inner product.

Reply:
Thanks for your review! I made a fix.